plugin/pkg/admission/security/podsecurity/admission.go

```diff
@@ -190,33 +187,22 @@ func (p *Plugin) Validate(ctx context.Context, a admission.Attributes, o admissi
 
 type lazyConvertingAttributes struct {
 	admission.Attributes
+}
 
-	convertObjectOnce sync.Once
-	convertedObject   runtime.Object
+func (l *lazyConvertingAttributes) GetObject() (runtime.Object, error) {
+	return convert(l.Attributes.GetObject())
+}
 
-	convertOldObjectOnce sync.Once
-	convertedOldObject   runtime.Object
+func (l *lazyConvertingAttributes) GetOldObject() (runtime.Object, error) {
+	return convert(l.Attributes.GetOldObject())
```

Comment:
this is expensive to do multiple times… we want to be sure we do it at most once

Reply:
same for GetObject

Reply:
See the PR description. I don't mind adding it back though.

> **Special notes for your reviewer:**
> I dropped the sync.Once from the conversion since we only ever get the Object once now, so I thought it was unnecessary overhead. I don't feel strongly about it, if you'd prefer to keep it.